html.md

```diff
@@ -49,7 +49,19 @@ updated: 2020-06-17
 |`<audio>`| It is used to embed sound content in the document. |` <audio controls> <source src="demo.mp3" type="audio/mpeg"> </audio>`|
 |`<video>`| It is used to embed video in the webpage. |` <video width="480" height="320" controls> <source src="demo_move.mp4" type="video/mp4"> </video>`|
 
+## HTML 5 Semantic Tags
```

Comment:
can you push this section to the bottom of the cheat sheet?

Reply:
Sure

Reply:
@onecompiler-ops done